modules/nf-core/repeatmodeler/builddatabase/tests/main.nf.test

```diff
@@ -0,0 +1,59 @@
+nextflow_process {
+
+    name "Test Process REPEATMODELER_BUILDDATABASE"
+    script "../main.nf"
+    process "REPEATMODELER_BUILDDATABASE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "repeatmodeler"
+    tag "repeatmodeler/builddatabase"
+
+    test("sarscov2-genome_fasta") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.versions).match("versions") },
+                { assert snapshot(process.out.db[0][1].collect { file(it).name }.sort().toString()).match("db") }
```

Comment:
So this is checking that a file with a name containing "db" exists? Were none of the files stable to check their md5sum?